src/asp/Tools/hiedr2mosaic.py

```diff
@@ -365,11 +365,12 @@ def get_ccd(path):
 def main():
     try:
         try:
-            usage = "usage: hiedr2mosaic.py [--help][--manual][--threads N][--keep][-m match] HiRISE-EDR.IMG-files\n  " + get_asp_version()
+            usage = "usage: hiedr2mosaic.py [--help][--manual][--threads N][--keep][-m match][-w bool] HiRISE-EDR.IMG-files\n  " + get_asp_version()
```

Comment:
Ug, our Python is so old, argparse now automatically writes a usage statement for you, but that clean-up is for another day.  Let's make this easier on the user and just add `[-w]` here.

Reply:
Ok, i'll leave it as it is.